### PR TITLE
Add Conformance classes

### LINKED ISSUE #192: Spec should be more specific about implementation details
from https://github.com/w3cping/privacy-request/issues/125#issuecomment-1849123834

> >  I think that was fairly clear. I'm pretty sure the WG means that an issuer MUST secure each VC (from the core data model spec), and one of the ways it MAY secure them is COSE. If the issuer does that, it SHOULD identify that fact using content types.

> Ok, that's good to hear. Yeah I could definitely see someone going "Hey I don't want to use COSE here for X, Y, and Z reason, but CBOR looks nice. I'll just use this unspecified content type that I made up and isn't defined anywhere and call it conformant with the VC-JOSE-COSE spec". Based on the usage of MAY and SHOULD here, they'd technically still be conformant even though it's wildly divergent from what's defined and I wanted to make sure that was called out to spec authors. This one should be as simple as just making the spec language a bit stronger. One example would be:

> "If an implementer wishes to use CBOR as a serialization format they MUST USE RFC9052 and MUST specify the content type as...".

> My language might still face tradeoffs here, but it at least remains narrowly defined to make it easy for someone to implement later. I'll open an issue on the spec to make sure people are aware of this concern.

### LINKED ISSUE #202: Conformance classes are not defined
The current specification does not define any conformance classes. This means that it is not possible for anyone to tell what is required from a conforming document or a conforming issuer implementation or a conforming verifier implementation. 

The specification needs to define conformance classes for at least two things: documents/serializations and software/implementations.

### PR DESCRIPTION
**Pull request by brentzundel**
This PR attempts to fix #202 and fix #192 

I am confident it will need adjustment after review and feedback.

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-jose-cose/pull/231.html" title="Last updated on Feb 20, 2024, 4:25 PM UTC (e17638d)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jose-cose/231/ad35710...brentzundel:e17638d.html" title="Last updated on Feb 20, 2024, 4:25 PM UTC (e17638d)">Diff</a>

### CODE REVIEW

**Comment by brentzundel** [comment]
I'm not sure what is causing the build error. @iherman any thoughts?

**Review by decentralgabe — APPROVED** [review]
excellent!

File: index.html
Comment (selfissued) [inline index.html:382]:
```suggestion
          A [=conforming CWT issuer implementation=] MUST use COSE_Sign1 as specified in [[RFC9052]] to secure this media type.
```

File: index.html
Comment (selfissued) [inline index.html:391]:
```suggestion
        A [=conforming CWT verifier implementation=] MUST use COSE_Sign1 as specified in [[RFC9052]] to verify [=conforming CWT documents=] that use this media type.
```

File: index.html
Comment (selfissued) [inline index.html:402]:
```suggestion
        A [=conforming CWT issuer implementation=] MUST use COSE_Sign1 as specified in [[RFC9052]] to secure this media type.
```

File: index.html
Comment (selfissued) [inline index.html:408]:
```suggestion
        A [=conforming CWT verifier implementation=] MUST use COSE_Sign1 as specified in [[RFC9052]] to verify [=conforming CWT documents=] that use this media type.
```

**Review by selfissued — APPROVED** [review]
Approved, after merging my suggestions.

File: index.html
Comment (selfissued) [inline index.html:194]:
```suggestion
    <section id="conformance" class="normative">
    <h2>Conformance Classes</h2>

```

Comment (selfissued) [reply index.html:194]: I think that adding the h2 will fix the build error.

**Comment by brentzundel** [comment]
@decentralgabe @selfissued I found the error!
I was adding a new conformance section, when it turns out there already was one. I added my new content to the existing section and now there are no build errors.
Let me know if you see any other needed changes.

**Pull request merged**
